modules/hpe_nimble_volume.py

```diff
@@ -0,0 +1,897 @@
+#!/usr/bin/python
+
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# author alok ranjan (alok.ranjan2@hpe.com)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+author:
+    - Alok Ranjan (@ranjanal)
+description: Manage volume on a Nimble Storage group
+module: hpe_nimble_volume
+options:
+    state:
```

Comment:
Do we know if these are rendered in alphabetical order? I can see that a lot of modules do that in the Ansible module index. Maybe we should sort alphabetically in the code too just to make it readable for our successors?

Reply:
yes they are always rendered in alphabetical order. 

Ok so if you see i have followed this convention.

almost all variables , barring our custom new variables required for modules , are listed in the same order as it is pressent in the API webpage.
https://qaserv.nimblestorage.com/artifacts/sm_default/latest-successful/html_docs/build/opt/html_docs/v1/volumes.html

a bit of manual effort is required if we have to keep it alphabetically.. is that sorting really required keeping in mind that the final ansible command automatically sorts it in alpha... order?